.github/workflows/release-publish.yaml

```
@@ -0,0 +1,56 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
```

Following our agreement that version tags will drop the `v` prefix, I believe this should be:
```suggestion
      - "[0-9]+.[0-9]+.[0-9]+"
```
([source](https://github.com/pytest-dev/pytest/blob/main/.github/workflows/deploy.yml#L8))